External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -2318,12 +2331,33 @@ static public void HandleAllClassesCompletion(ScintillaControl Sci, string tail,
                     known = found;
                 }
             }
-
+            
             if (ASContext.Context.Features.hasDelegates && !ASContext.Context.CurrentClass.IsVoid())
```

Comment:
Adding delegate info and generics info both depend on being inside a valid class.